packages/accordion/src/LionAccordion.js

```diff
@@ -129,6 +148,7 @@ export class LionAccordion extends LitElement {
       <div class="accordion">
         <slot name="invoker"></slot>
         <slot name="content"></slot>
+        <slot name="accordion"></slot>
```

Comment:
Am i correct that this slot should not be used by our users, so its basically a protected thing. Maybe call it `_accordion`?

Reply:
Updated.